articles/data-factory/continuous-integration-delivery-sample-script.md

```diff
@@ -24,14 +24,14 @@ Install the latest Azure PowerShell modules by following instructions in [How to
 >[!WARNING]
 >Make sure to use **PowerShell Core** in ADO task to run the script
 
-## Pre- and post-deployment script
+## Pre- and post-deployment script 
 The sample scripts to stop/ start triggers and update global parameters during release process (CICD) are located in the [Azure Data Factory Official GitHub page](https://github.com/Azure/Azure-DataFactory/tree/main/SamplesV2/ContinuousIntegrationAndDelivery).
 
 > [!NOTE]
 > Use the [PrePostDeploymentScript.Ver2.ps1](https://github.com/Azure/Azure-DataFactory/blob/main/SamplesV2/ContinuousIntegrationAndDelivery/PrePostDeploymentScript.Ver2.ps1) if you would like to turn off/ on only the triggers that have been modified instead of turning all triggers off/ on during CI/CD.
 
 
-## Script execution and parameters
+## Script execution and parameters - Classic Release
```

Comment:
Changing this heading is causing a link to break in another document. You'll need to fix that before we can merge this.